cpp/ql/src/qlpack.yml

```diff
@@ -1,6 +1,6 @@
 name: codeql/cpp-queries
 version: 0.0.7-dev
-groups: cpp
+groups: cpp,queries
```

Comment:
Why is `groups` not a list? Using a comma-separated string instead of an array-type feels weird .